wordCount.rb

```diff
@@ -0,0 +1,33 @@
+class WordCounter
+  attr_accessor :file, :input, :output
+  def initialize(file)
+    @file = file
+    @input = ""
+    @output = {}
+  end
+
+  def parse
+    File.foreach(@file) {|x| count(x)}
+  end
+
+  def print
+    @output = @output.sort_by {|key, value| value}
+    @output.each {|key, value| puts "#{value} - #{key}"}
+  end
+
+  private
+  def count(input)
+    input = input.split
+    input.each do |word|
+      if @output[word] == nil
```

Comment:
Ooooh, so if I set @output =  Hash.new(0) then all values for key-pairs default to 0? That'd be nifty, yep I see what you are saying, cool.